### PR TITLE
Rewrite rarbg provider to remove the nonsense and conform to other pr…

### PR DESCRIPTION
**Pull request by miigotu**
…ovider formats

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by fernandog** [comment]
@miigotu

We need format=json_extended for size

File: sickbeard/providers/rarbg.py [hunk 1]
```diff
@@ -47,68 +40,51 @@ def __init__(self):
...
         if not self.login():
             return results
 
+        search_params = {
+            'app_id': 'sickrage2',
+            'categories': 'tv',
+            'seeders': int(self.minseed),
+            'leechers': int(self.minleech),
+            'limit': 100,
+            'format': 'json_extended',
```
Comment (miigotu) [inline sickbeard/providers/rarbg.py:83]: @fernandog only for search_params, not login_params.

**Comment by fernandog** [comment]
@miigotu 

something wrong with tests or rarbg?

```
======================================================================
ERROR: test_manual_Game_of_Thrones_121361_Rarbg (__main__.SearchTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "search_tests.py", line 120, in do_test
    items = cur_provider.search(search_strings)  # pylint: disable=protected-access
  File "/home/osmc/SickRage/sickbeard/providers/rarbg.py", line 80, in search
    'seeders': int(self.minseed),
TypeError: int() argument must be a string or a number, not 'NoneType'

```

**Pull request merged**